astro/src/content/docs/get-started/start-here/index.mdx

```diff
@@ -0,0 +1,165 @@
+
+* **pre-built authentication workflows** and integrations accelerate application delivery
+* **secure storage of user credentials and profile data**, wherever you want (you host or we host)
+* **new functionality and bugfixes** from the security experts on the FusionAuth team, available to your users when you upgrade
+* **the same experience** and application whether you develop locally, use FusionAuth in your CI/CD system, or run it in prod
+* fine grained control of the version of FusionAuth you use; **upgrades happen on your schedule, not ours**
+
+This guide will walk you through standing up a sample, pre-configured application that uses FusionAuth for login.
+
+You'll also add the ability to log a user in using an emailed magic link.
+
+## Overview of Using FusionAuth
+
+To get started with FusionAuth, you need to do three things:
+
+* Launch FusionAuth
+* Delegate authentication
+* Add functionality to your application
+
+<StartDiagram alt="Launch, then delegate, then add functionality." />
+
+Let's walk through an example where you'll add a Login With Google button to an application with no code changes required.
+
+## Prerequisites
+
+* git
+* Docker (if you want to follow the Docker instructions)
+* a GitHub account (if you choose to use GitPod instead)
+* node (this was tested with `v22.12.0`
+
+## Launch FusionAuth
+
+FusionAuth is downloadable software, so you need to run it somewhere. For this guide, you can either:
+
+* Use Docker to install and run FusionAuth and the application
+* Use GitPod to run FusionAuth in the cloud in your GitPod account, with no software install needed.
+
+Expand the section you want to follow and get started.
+
+<details>
+<summary>Docker</summary>
+<DockerInstallAndStart codeRoot={frontmatter.codeRoot} kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"}/>
+</details>
+
+<details>
+<summary>GitPod</summary>
+<GitPodStart codeRoot={frontmatter.codeRoot} kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"}/>
+</details>
+
+## Delegate Authentication
+
+Once you have FusionAuth up and running, you need to set up your application or applications to delegate to FusionAuth. For this guide, there's an example 'Start Here' application which is already set up to do so. Let's look at that configuration and then start it up so you can log in.
+
+<details>
+<summary>Open the Admin Interface</summary>
+<DelegateStep1 />
+</details>
+
+<DelegateStep2 />
+
+<details>
+<summary>Open the Application URL in your Browser and Login</summary>
+<DelegateStep3 />
+</details>
+
+### What Did You Just Do?
+
+You looked at some FusionAuth configuration, started a JavaScript application and checked out some cod, and logged in to the application using FusionAuth.
+
+## Add Functionality
+
```

Comment:
This section was missing from the gitpod installed instructions. Was that intentional, or is that a function of fusionauth.io not having this content live?

Reply:
I added this for the launch section.